tests/countries/test_finland.py

```diff
@@ -165,6 +167,152 @@ def test_all_saints_day(self):
             "1957-11-01",
         )
 
+    def _test_unofficial_holiday(self, name, month_and_day, year_of_adoption):
+        self.assertHolidayName(
+            name,
+            self.unofficial_holidays,
+            (f"{year}-{month_and_day}" for year in range(year_of_adoption, 2031)),
+        )
+        self.assertNoHolidayName(name, self.unofficial_holidays, year_of_adoption - 1)
+
+    def test_runeberg_day(self):
+        self._test_unofficial_holiday("Runebergin päivä", "02-05", 1854)
+
+    def test_kalevala_day(self):
+        self._test_unofficial_holiday(
+            "Kalevalan päivä, suomalaisen kulttuurin päivä", "02-28", 1860
+        )
+
+    def test_minna_canth_day(self):
+        self._test_unofficial_holiday("Minna Canthin päivä, tasa-arvon päivä", "03-19", 2007)
+
+    def test_agricola_day(self):
+        self._test_unofficial_holiday("Mikael Agricolan päivä, suomen kielen päivä", "04-09", 1978)
+
+    def test_veterans_day(self):
+        self._test_unofficial_holiday("Kansallinen veteraanipäivä", "04-27", 1987)
+
+    def test_europe_day(self):
+        self._test_unofficial_holiday("Eurooppa-päivä", "05-09", 2019)
+
+    def test_mothers_day(self):
+        self.assertHolidayName(
+            "Äitienpäivä",
+            self.unofficial_holidays,
+            "1918-05-12",
+            "1919-05-11",
+            "2020-05-10",
+            "2024-05-12",
+        )
+        self.assertNoHoliday(
+            self.unofficial_holidays,
+            "1917-05-13",
+        )
+
+    def test_finnish_identity_day(self):
+        self._test_unofficial_holiday("J.V. Snellmanin päivä, suomalaisuuden päivä", "05-12", 1952)
+
+    def test_remembrance_day(self):
+        self.assertHolidayName(
+            "Kaatuneitten muistopäivä",
+            self.unofficial_holidays,
+            "1977-05-15",
+            "1978-05-21",
+            "1985-05-19",
+            "2024-05-19",
+            "2025-05-18",
+        )
+        self.assertNoHoliday(
+            self.unofficial_holidays,
+            "1976-05-16",
+            "1975-05-18",
+        )
+
+    def test_defence_forces_day(self):
```

Comment:
```suggestion
    def test_defense_forces_day(self):
```

Reply:
omg 🙈

Reply:
hm, no, actually it is defen*c*e with a c: https://en.wikipedia.org/wiki/Finnish_Defence_Forces